src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -165,7 +161,7 @@ private function getMetadata($type, $entity)
                     return false;
                 }
 
-                return !$param->getClass() || $param->getClass()->getName() !== 'Twig_Environment';
+                return !$param->getClass() || !in_array($param->getClass()->getName(), array('Twig_Environment', 'Twig\Twig'));
```

Comment:
Using `class_alias`, the `ReflectionClass` refers to the original class. Checking against `Twig\Twig` is not necessary.
https://3v4l.org/QH2KQ